tests/unit/test_workflows.py

```diff
@@ -2268,6 +2268,8 @@ async def test_workflow_runs_template_for_each(
             name="template_action",
             namespace="testing",
             display_group="Testing",
+            doc_url=None,
+            author=None,
```

Comment:
See comment below, you shouldn't need this after pydantic defaults are properly specified